spec/feature_flagger/storage/redis_spec.rb

```diff
@@ -3,63 +3,103 @@
 RSpec.describe FeatureFlagger::Storage::Redis do
   let(:redis)   { FakeRedis::Redis.new }
   let(:storage) { described_class.new(redis) }
-  let(:key)   { 'foo' }
-  let(:value) { 'bar' }
-  let(:global_key) { 'released_features' }
+  let(:feature_key)   { 'account:email_marketing:whitelabel' }
+  let(:resource_id)   { '1' }
+  let(:resource_name) { 'account' }
+  let(:global_key)    { 'released_features' }
+  let(:resource_key) do
+    FeatureFlagger::Storage::RedisKeys.resource_key(
+      FeatureFlagger::Storage::Redis::RESOURCE_PREFIX,
+      resource_name,
+      resource_id,
+    )
+  end
 
   context do
     before do
       redis.flushdb
     end
 
-    describe '#has_value?' do
-      context 'value is stored for given key' do
-        before { redis.sadd(key, value) }
-        it { expect(storage).to have_value(key, value) }
+    describe '#has_values?' do
+      context 'resource_id is stored for given feature_key' do
+        before { redis.sadd(feature_key, resource_id) }
```

Comment:
I think we could use the `storage.add` method, what do you think?

Reply:
make sense, done!